vm/vm/src/tevm.spec.ts

```diff
@@ -305,6 +305,51 @@ describe('Tevm should create a local vm in JavaScript', () => {
 			expect(res.id).toBe(1)
 			expect(res.method).toBe('tevm_putContractCode')
 		})
+
+		it('Should fail a evm call request if the file is too large', async () => {
+			const tevm = await createTevm()
+			const address1 = '0x1f420000000000000000000000000000000000ff'
+
+			/*
+				Simple bytecode to deploy a large contract
+				`62` PUSH4, `FFFFFF` value being pushed, `60` PUSH1, `00` value, `F3` return
+				https://github.com/ethereumjs/ethereumjs-monorepo/blob/a0ef459e26f6a843d67bb2142977b67359109839/packages/evm/test/runCall.spec.ts#L543
+			*/
+			const data = '0x62FFFFFF6000F3'
+
+			const res = await tevm.request({
+				id: 1,
+				method: 'tevm_call',
+				jsonrpc: '2.0',
+				params: {
+					caller: address1,
+					data
+				},
+			})
+			expect(res.result.execResult.exceptionError?.error.toString()).toBe("code size to deposit exceeds maximum code size")
```

Comment:
Is casting the error message to a string a good idea?

Reply:
```suggestion
			expect(res.result.execResult.exceptionError?.error)).toThrowErrorMatching("code size to deposit exceeds maximum code size")
```
It's fine but I think vitest can also do things like this? I tend to not care too much about stylistic things in tests though

Reply:
I found that ethereumjs uses an enum to compare in their tests
https://github.com/ethereumjs/ethereumjs-monorepo/blob/76b74d0343f5c595f0107032d1daec61fbc0c72d/packages/evm/src/exceptions.ts#L1

So I am using that

Reply:
Great find! I strongly type errors using effect.ts. Later when we clean up the vm strongly typing the errors this will make it really easy